app/assets/stylesheets/style.scss

```diff
@@ -62,7 +62,17 @@ It was originally created by Caroline Hadilaksono: http://www.hadilaksono.com
 .map-list div:nth-child(4n+1){
   clear: left;
 }
-
+.e2_48 { 
+	width:250.7894592285156px;
+	height:160.3662719726562px;
+	/* position:absolute; */
+	left:-0.00002248663622594904px;
+	top:-88.56105041503906px;
+	background-image:url("https://3000-publiclab-mapknitter-1f5i6spevl6.ws-us39.gitpod.io/assets/image-cd28b9101644e1fb24be3adb6baf2d8717d366c3df526f4ec8436b7efba7a73c.png");
```

Comment:
Kindly check if the bg-image url will work remotely.

Reply:
@cesswairimu Thanks for pointing it out! 💯  I had made the required changes, now the bg-image URL will work remotely 😄. Kindly review it. Thank You!